tests/test_accelerator.py

```diff
@@ -762,3 +762,64 @@ def test_save_model_with_stateful_dataloader(self, use_safetensors, tied_weights
             assert torch.allclose(original_linear1, new_linear1)
             assert torch.allclose(original_batchnorm, new_batchnorm)
             assert torch.allclose(original_linear2, new_linear2)
+
+    @require_cuda
```

Comment:
we need a require_transformers decorator if you are going to import transformers here. Otherwise, we can try to not use it and just save the model with 

```
checkpoint = os.path.join(tmp_dir, "pt_model.bin")
torch.save(model.state_dict(), checkpoint)
```